src/configureStore.js

```diff
@@ -1,11 +1,19 @@
 import { createStore, applyMiddleware } from 'redux';
+import createSagaMiddleware from 'redux-saga';
 import createLogger from 'redux-logger';
 
 import rootReducer from './reducers/';
+import sagas from './sagas/index';
```

Comment:
Sagas might fit better into the world of redux which we already are engaged to, but I'm going to ask anyway.
Did you consider [axios](https://github.com/mzabriskie/axios)?